dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/monitor/DatadogRumMonitor.kt

```diff
@@ -425,6 +453,27 @@ internal class DatadogRumMonitor(
         updateAndSendView { it.incrementUserActionCount() }
     }
 
+    private fun registerUserAction(
+        action: String,
+        attributes: Map<String, Any?>
+    ) {
+        // invalidate the previous pending closeable action flag in order to make it ready for
+        // being closed
+        isWaitingForManualClosing = false
```

Comment:
it seems the only time `sendUnclosedAction` is called, `isWaitingForManualClosing` is already set to false (either false, or the function can only be called if it's already false). 

so in theory you're not changing anything by calling it here.

Reply:
Hmm...not exactly...it is being called also from `getActiveActionId` and there you have this condition `isWithinActionScope() || isWaitingForManualClosing` which doesn't necessarily means  that this flag is false.